resources/mysql/mysql.go

```diff
@@ -17,7 +17,12 @@ import (
 	_ "github.com/go-sql-driver/mysql"
 )
 
+//go:generate curl -O https://cdn.mysql.com//Downloads/MySQL-8.0/mysql-8.0.28-macos11-arm64.tar.gz -z mysql-8.0.28-macos11-arm64.tar.gz
```

Comment:
Download only if the file doesn't exist?

Reply:
yes, -z request a file that has been modified later than the given file's mtime. so that it will not download repeated.
see https://man7.org/linux/man-pages/man1/curl.1.html#:~:text=%2Dz%2C%20%2D%2Dtime%2Dcond,will%20be%0A%20%20%20%20%20%20%20%20%20%20%20%20%20%20used.

Reply:
When download failed in the middle, the whole process will be stuck on the in-complete file till it's manually removed. We can change the process to.
1) check file existence;
2) download to 1.gz.tmp;
3) move 1.gz.tmp 1.gz